### PR TITLE
add : input component

### PR DESCRIPTION
**Pull request by Varun-28**
Added Input component -> one simple and one validation type
input2
textbox1

### CODE REVIEW

File: css/component_styles/input.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/* ---------- input css -------- */
+
+.input-wrapper,
+.input-group>* {
+    display: block;
+}
+
+.input-group p {
+    font-size: 12px;
```
Comment (shivampandey0) [inline css/component_styles/input.css]: use rem for font-size because you need to make your text responsive.

**Review by shivampandey0 — COMMENTED** [review]
LGTM

**Review by Hamdrive — COMMENTED** [review]
Looks good, no changes

File: css/component_styles/input.css (hunk 1 above)
Comment (Varun-28) [reply css/component_styles/input.css]: I have changed It to rem 👍

**Pull request merged**